rfc/0070-normalise-mod-ids.md

```diff
@@ -0,0 +1,49 @@
+# Normalise Mod ID Separators
+
+## Summary
+
+Treat dashes and underscores as the same for mod id comparison purposes.
+
+## Motivation
+
+An occasional cause for confusion is when two different mods only differ by underscores (`_`) and dashes(`-`) in their mod-ids. Since those characters aren't usually used to discriminate between mods, I propose we consider them as the same for the purposes of mod resolution and solving.
```

Comment:
```suggestion
An occasional cause for confusion is when two different mods only differ by underscores (`_`) and dashes(`-`) in their mod IDs. Since those characters aren't usually used to discriminate between mods, I propose we consider them as the same for the purposes of mod resolution and solving.
```